src/com/lftechnology/java/training/dipak/multiples/Multiples.java

```diff
@@ -0,0 +1,104 @@
+package com.lftechnology.java.training.dipak.multiples;
+
+import java.util.Scanner;
+
+import java.util.logging.Logger;
+
+/*	
+ * If we list all the natural numbers below 10 that are multiples of 3 or 5, we get 3, 5, 6 and 9. The sum of these multiples is 23.
+
+ *Find the sum of all the multiples of 3 or 5 below 1000.
+*/
+
+public class Multiples {
+	private final static Logger LOGGER = Logger.getLogger(Multiples.class.getName());
+	int multiple[];	
+	int index=0;
+	int range;		
+	Multiples(int a){
+		range=a;
+		multiple=new int[range];	

```

Comment:
why is the variable 'a' is being assigned to range variable  . you could have taken range as a parameter
 i.e arrayOfMultiples( int range).
Also use meaningful variables (dont use 'a' as variable) and method name